packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -199,8 +202,9 @@ export class TokenRatesController extends StaticIntervalPollingControllerV1<
       getNetworkClientById,
       chainId: initialChainId,
       ticker: initialTicker,
-      selectedAddress: initialSelectedAddress,
-      onPreferencesStateChange,
+      selectedAccountId,
+      getInternalAccount,
+      onSelectedAccountChange,
```

Comment:
Similar comment as above, but in addition, instead of continuing with the callback pattern is it possible to listen to the `AccountsController:selectedAccountChange` event instead?